boards/riscv/tl321x/tl3218x-pinctrl.dtsi

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2023 Telink Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/pinctrl/tl321x-pinctrl.h>
+
+&pinctrl {
+	/* Set pad-mul-sel register value.
+	 * Note: Pins functions below (pinmux = <...>) depend on this value.
+	 */
+	pad-mul-sel = <1>;
+
+	/* UART0: TX(PB2), RX(PB3), RTS(PB4), CTS(PB6) */
+
+	uart0_tx_pb2_default: uart0_tx_pb2_default {
+		pinmux = <B9x_PINMUX_SET(B9x_PORT_B, B9x_PIN_2, B92_FUNC_UART0_TX)>;
```

Comment:
B9x , B92?

Reply:
Damien will make these changes in another PR.